deployments/stat159/image/infra-requirements.txt

```diff
@@ -1,34 +1,9 @@
-# WARNING: Original source at scripts/infra-packages/requirements.txt
-# PLEASE DO NOT EDIT ELSEWHERE
-# After editing scripts/infra-packages/requirements.txt, please run
-# scripts/infra-packages/sync.bash.
+# A few packages not available on conda-forge. The rest of the environment is 
```

Comment:
Ah, so infra-requirements.txt is managed globally with https://github.com/berkeley-dsep-infra/datahub/tree/staging/scripts/infra-packages - so we can keep it in sync across all the images. It would be great if we can leave it unmodified, as the next run of the script will mangle all these changes.

Reply:
Yup, I saw that and was sure it would require some iteration, thx...

I guess the question is - given you're moving more to conda packages, how would you prefer to handle this idea of defining the complete env in a conda `.yml` file for each course, while having this global one?

I'm happy to iterate on ideas, but I didn't want to mess with the global file to fit the packages I was after for this class...

Reply:
@fperez yeah, we haven't figured out how to move this through quite yet.

For now, do you think you can just leave out packages that exist in the global file from the stat159 environment.yml? I think the global file should be uptodate on versions.